### PR TITLE
[16.0][MIG] sale_order_lot_generator

### PR DESCRIPTION
**Pull request by florian-dacosta**
depends on https://github.com/OCA/sale-workflow/pull/2282

### CODE REVIEW

**Comment by florian-dacosta** [comment]
@bealdav @sebastienbeau @newtratip  Ready for review

File: sale_order_lot_generator/models/sale_order_line.py [hunk 1]
```diff
@@ -21,21 +21,21 @@ def _prepare_vals_lot_number(self, index_lot):
     def create_prodlot(self, index_lot=1):
```
Comment (bealdav) [inline sale_order_lot_generator/models/sale_order_line.py]: To be consistent I supose create_prodlot should be rename without prod

Comment (florian-dacosta) [reply sale_order_lot_generator/models/sale_order_line.py]: Done

Comment (florian-dacosta) [reply sale_order_lot_generator/models/sale_order_line.py]: Done

Comment (rvalyi) [reply sale_order_lot_generator/models/sale_order_line.py]: a detail, but if you changed create_prodlot to create_lot, I would change also the generate_prodlot method to generate_lot. If you do it I approve right now...

Comment (florian-dacosta) [reply sale_order_lot_generator/models/sale_order_line.py]: Done

**Comment by florian-dacosta** [comment]
@bealdav @rvalyi changes have been done, feel free to update your reviews!

**Review by rvalyi — APPROVED** [review]

**Review by bealdav — APPROVED** [review]

**Comment by florian-dacosta** [comment]
PR is quite old and approved, can any PSC merge this one ?
@sebastienbeau @rvalyi @rousseldenis

**Pull request merged**